src/app/components/home/task-container/add-task/add-task.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+import { TaskFactoryService } from 'src/app/services/task-factory.service';
+import { TaskService } from 'src/app/services/task.service';
+import { UtilityService } from 'src/app/services/utility.service';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+
+@Component({
+  selector: 'app-add-task',
+  templateUrl: './add-task.component.html',
+  styleUrls: ['./add-task.component.scss']
+})
+export class AddTaskComponent {
+
+  faDelete = faTrash;
```

Comment:
```suggestion
export class AddTaskComponent {
  faDelete = faTrash;
```

Reply:
Is there any reason to use faDelete property?

Reply:
I have stored the fontawesome faTrash icon in this property and used it in html to show the icon.